tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -511,6 +511,60 @@ compose.desktop {
 }
 ```
 
+## File associations
```

Comment:
Thanks!

Could you move the docs to another PR to `support/1.7.0` branch? It will be reviewed by tech writers and be merged at the release date)

Reply:
Done